lib/tasks/client_generate.rake

```diff
@@ -0,0 +1,75 @@
+#
+# Usage: bundle exec rake client:generate
+#        bundle exec rake client:generate[/alternate_ruby_client_dir]
+#
+class ClientGenerator
+  require 'json'
+  require 'uri'
+
+  VERSION = "3.3.4".freeze
```

Comment:
Its unfortunate that http://central.maven.org/maven2/org/openapitools/openapi-generator-cli/ doesn't have a `latest` directory